src/main/groovy/com/okta/test/mock/application/ApplicationTestRunner.groovy

```diff
@@ -116,8 +116,8 @@ abstract class ApplicationTestRunner extends HttpMock {
     }
 
     ApplicationUnderTest getApplicationUnderTest(String scenarioName) {
-
-        Config config = new Yaml().loadAs(getClass().getResource( '/testRunner.yml' ).text, Config)
+        File yamlFile = new File(System.getProperty("config"))
+        Config config = new Yaml().loadAs(new FileInputStream(yamlFile), Config)
```

Comment:
We should probably check for both. If the `config` property is set use that, otherwise attempt to load the the file from the classpath.

Reply:
@bdemers - have updated to support both options